task.sql

```diff
@@ -1 +1,6 @@
--- write your code for database user creation here
\ No newline at end of file
+CREATE USER 'webappuser'@'%' IDENTIFIED BY 'P@ssw0rd';
+GRANT INSERT, UPDATE, DELETE, SELECT ON ShopDB.* TO 'webappuser'@'%';
+
+CREATE USER 'deploymentuser'@'%' IDENTIFIED BY 'P@ssw0rd';
+GRANT ALL ON ShopDB.* TO 'deploymentuser'@'%';
+ 
```

Comment:
![image](https://github.com/user-attachments/assets/59cd450d-a9d6-4aab-9b59-cdfb50943368)
The last line isn't empty so EoF didn't display correct

Reply:
Removed